scipy/stats/_distn_infrastructure.py

```diff
@@ -3208,38 +3197,55 @@ def fun(x):
         else:
             invfac = 1.0
 
-        tot = 0.0
-        low, upp = self._ppf(0.001, *args), self._ppf(0.999, *args)
-        low = max(min(-suppnmin, low), lb)
-        upp = min(max(suppnmin, upp), ub)
-        supp = np.arange(low, upp+1, self.inc)  # check limits
-        tot = np.sum(fun(supp))
-        diff = 1e100
-        pos = upp + self.inc
-        count = 0
+        x0 = self.ppf(0.5, *args, loc=loc)
```

Comment:
Is `loc` correct here? Do we integrate the original or the standardized distribution (with loc=0)?
I thought it was the standardized.